aspnetcore/fundamentals/minimal-apis/7.0-samples/todo-group2/Program.cs

```diff
@@ -0,0 +1,38 @@
+using Microsoft.EntityFrameworkCore;
+using MinApiRouteGroupSample;
+
+var builder = WebApplication.CreateBuilder(args);
+
+// Add services to the container.
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+builder.Services.AddDbContext<TodoGroupDbContext>(options =>
+{
+    var path = Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
+    options.UseSqlite($"Data Source={Path.Join(path, "WebMinRouteGroup.db")}");
+});
+
+
+var app = builder.Build();
+
+using var scope = app.Services.CreateScope();
+var db = scope.ServiceProvider.GetService<TodoGroupDbContext>();
+db?.Database.MigrateAsync();
+
+app.MapGroup("/public/todos")
+    .MapTodosApi()
+    .WithTags("Public Todo Endpoints");
+
+app.MapGroup("/private/todos")
+    .MapTodosApi()
+    .RequireAuthorization();
+
+// Configure the HTTP request pipeline.
+if (app.Environment.IsDevelopment())
```

Comment:
Can you move this to L18?

Reply:
@captainsafia 
> Can you move this to L18?
Is this what you want?

```suggestion
var app = builder.Build();

app.MapGroup("/private/todos")
    .MapTodosApi()
    .RequireAuthorization();

using var scope = app.Services.CreateScope();
var db = scope.ServiceProvider.GetService<TodoGroupDbContext>();
db?.Database.MigrateAsync();

app.MapGroup("/public/todos")
    .MapTodosApi()
    .WithTags("Public Todo Endpoints");

// Configure the HTTP request pipeline.
if (app.Environment.IsDevelopment())
```